#file: RemoveWriter.srv
#author: Joao Messias (jmessias@isr.ist.utl.pt)
#	 Jose Joaquin Acevedo (jacevedo@isr.ist.utl.pt)
#
#This is the description file for the service
#of removing (unregistering) a SAM Writer.
#
#The request consists of the name of the target slot and the name of the Writer to remove (this can be called by someone
#else other than the Writer itself).
string slot_name
string writer_name
---
#The reply contains a status flag (success / failure).
bool success
